mfr/extensions/video/render.py

```diff
@@ -13,7 +14,14 @@ class VideoRenderer(extension.BaseRenderer):
         ]).get_template('viewer.mako')
 
     def render(self):
-        return self.TEMPLATE.render(base=self.assets_url, url=self.url)
+        self_url = urlparse(self.url)
```

Comment:
This problem could more generally also affect other renderers that directly serve the file content, such as PDB files. Might be worth checking for similar edge case bugs?